corehq/apps/email/views.py

```diff
@@ -0,0 +1,56 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, redirect
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+
+from corehq import toggles
+from corehq.apps.domain.decorators import domain_admin_required, login_and_domain_required
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.email.forms import EmailSMTPSettingsForm
+from corehq.apps.email.models import EmailSettings
+
+
+@login_and_domain_required
+def default(request, domain):
+    from corehq.messaging.scheduling.views import MessagingDashboardView
+    return HttpResponseRedirect(reverse(MessagingDashboardView.urlname, args=[domain]))
```

Comment:
Where's this view for?  Should it redirect to the `EmailSMTPSettingsView` instead?  Or why not use that view as the default directly?

Reply:
This redirects to messaging dashboard view. Kept this behavior similar to SMS app where it defaults to `MessagingDashboardView`. Changed `urls.py` to call this view directly in the base email url.

![image](https://github.com/dimagi/commcare-hq/assets/14318173/2f405d5f-7e58-44eb-a9f7-ff8c7072b923)